scripts/checkChecksums.ts

```diff
@@ -0,0 +1,63 @@
+import { execSync } from "child_process";
+import { dirname, join } from "path";
+import { exit } from "process";
+
+const a: string[] = execSync(
+  "grep -Rw docs/src/ -A 1 -e 'load:' | sed -E 's/--//'"
+).toString().split('\n').filter(line => line !== '');
+
+let errorString: string = '';
+
+let index: number = 0;
+while (index < a.length) {
+  const currentLine: string = a[index];
+  const docsFile: string = currentLine.split(':')[0];
+  const r1: RegExpMatchArray | null = currentLine.match(
+    /load: (.+)/
+  );
+  if (r1) {
+    const [ _fullMatch, programPath ] = r1;
+    const nextLine: string = a[index + 1];
+    const r2: RegExpMatchArray | null = nextLine.match(
+      /md5: (.+)/
+    );
+    if (r2) {
+      const topDir: string = dirname(dirname(__filename));
+      const actualMD5WithNewline: string = execSync(`md5sum ${
```

Comment:
Don't run the `md5sum` program. Node already has the `crypto` module with the `createHash` function (https://nodejs.org/api/crypto.html#cryptocreatehashalgorithm-options) and `'MD5'` is one of the options.

Reply:
Oh, neat! I didn't know `node` had that. Thanks for the tip!